index.html

```diff
@@ -33,6 +33,274 @@
+      {{MediaDevices/ScreenCaptureMediaStreamTrack}} extends {{MediaStreamTrack}} and
+      corresponds to a <a data-cite="screen-capture#dfn-monitor">monitor</a>.
+      It additionally provides a connection to the window placement API [[!window-placement]]
+      with the {{MediaDevices/ScreenCaptureMediaStreamTrack/screenDetailed}} function with which metadata about the
+      captured <a data-cite="screen-capture#dfn-monitor">monitor</a> can be retrieved.
+      <pre class="idl">
+        [Exposed=Window]
+        interface ScreenCaptureMediaStreamTrack : MediaStreamTrack {
+          ScreenDetailed screenDetailed();
+        };
+      </pre>
+
+      <dl data-link-for="ScreenCaptureMediaStreamTrack" data-dfn-for="ScreenCaptureMediaStreamTrack" class="methods">
+        <dt>
+          <dfn>screenDetailed</dfn>
+        </dt>
+        The {{screenDetailed()}} method returns a {{ScreenDetailed}} object [[!window-placement]]
+        that corresponds to the <a data-cite="screen-capture#dfn-monitor">monitor</a> that
+        is captured by the {{ScreenCaptureMediaStreamTrack}} object.
+      </dl>
+    </section>
+
+    <section>
+      <h2>Misc</h2>
+      This specification extends the sections
+      <a data-cite="screen-capture#hidden-display-surfaces">Closed and Minimized Display Surfaces</a>, and
+      <a data-cite="screen-capture#deviceId">Device Identifiers</a>
+      of {{MediaDevices/getDisplayMedia()}} to include {{MediaDevices/getAllScreenMedia()}}.
+    </section>
+  </section>
+  <section id="privacyindicatorsrequirements">
+    <h2>
+      Privacy Indicator Requirements
+    </h2>
+    <p>
+      References in this specification to [[\devicesLiveMap]], and
+      [[\devicesAccessibleMap]] refer to the
+      definitions already created to support Privacy Indicator Requirements for
+      {{MediaDevices/getDisplayMedia()}}.
+    </p>
+    <p>
+      This specification extends the <a data-cite="screen-capture#privacy-indicator-requirements">
+      Privacy Indicator Requirements</a> of
+      {{MediaDevices/getDisplayMedia()}} to include {{MediaDevices/getAllScreenMedia()}}.
+      In addition to these requirements, <a href="#dfn-user-agent">user agents</a> MUST
+      ensure that privacy indicators are visible at all times and that dismissal of the indicators
+      must not be persisted. The privacy indicators must inform about the <a data-cite="url#concept-url-origin">origin</a>
+      of the application capturing the screens. The indicators MUST clearly inform the user that the
+      <a data-cite="screen-capture#dfn-monitor">monitors</a> are captured (and potentially recorded).
+      The indicators MUST remain active for at five seconds, even if the capturing is ended earlier.
+    </p>
+    <p>
+      The <a href="#dfn-user-agent">user agents</a> MUST provide the user with the means to look up
+      whether {{MediaDevices/getAllScreenMedia()}} is enabled and to learn about the implications
```

Comment:
Do you mean "whether any origin is permitted to call gASM" or something like that?

Should we perhaps not only allow this lookup - which most users would never undertake - but rather force a persistent permission? If I am using a machine that could start recording me at any moment, I want to know ahead of time, not after the recording starts.

Reply:
wouldn't this prevent the owner from changing (and in particular restricting) the allowlist? the owner of the user account / device will always know ahead of time what allowlist they are setting.

Reply:
When I said "persistent permission", I meant "persistent notification." Sorry for the confusion; I hope the entire message is much more coherent given that change.